src/VolumeLogger.js

```diff
@@ -59,6 +59,295 @@ module.exports = class VolumeLogger {
+                            bytesToNetworkPerSec: metricsReport.metrics['broker/publisher'].bytes.rate,
+                            messagesFromNetworkPerSec: 0,
+                            bytesFromNetworkPerSec: 0,
+                        },
+                        network: {
+                            avgLatencyMs: metricsReport.metrics.node.latency.rate,
+                            bytesToPeersPerSec: metricsReport.metrics.WebRtcEndpoint.outSpeed.rate || 0,
+                            bytesFromPeersPerSec: metricsReport.metrics.WebRtcEndpoint.inSpeed.rate || 0,
+                            connections: metricsReport.metrics.WebRtcEndpoint.connections || 0
+                        },
+                        storage: {
+                            bytesWrittenPerSec: (metricsReport.metrics['broker/cassandra']) ? metricsReport.metrics['broker/cassandra'].writeBytes : 0,
+                            bytesReadPerSec: (metricsReport.metrics['broker/cassandra']) ? metricsReport.metrics['broker/cassandra'].readBytes : 0,
+                        },
+
+                        startTime: metricsReport.startTime,
+                        currentTime: metricsReport.currentTime,
+                        timestamp: metricsReport.currentTime,
+                    }
+
+                    const dayReport = {
+                        // peerId: brokerAddress,
+                        peerName: metricsReport.peerId,
+                        broker: {
+                            messagesToNetworkPerSec: metricsReport.metrics['broker/publisher'].messages.rate,
+                            bytesToNetworkPerSec: metricsReport.metrics['broker/publisher'].bytes.rate,
+                            messagesFromNetworkPerSec: 0,
+                            bytesFromNetworkPerSec: 0,
+                        },
+                        network: {
+                            avgLatencyMs: metricsReport.metrics.node.latency.rate,
+                            bytesToPeersPerSec: metricsReport.metrics.WebRtcEndpoint.outSpeed.rate || 0,
+                            bytesFromPeersPerSec: metricsReport.metrics.WebRtcEndpoint.inSpeed.rate || 0,
+                            connections: metricsReport.metrics.WebRtcEndpoint.connections || 0
+                        },
+                        storage: {
+                            bytesWrittenPerSec: (metricsReport.metrics['broker/cassandra']) ? metricsReport.metrics['broker/cassandra'].writeBytes : 0,
+                            bytesReadPerSec: (metricsReport.metrics['broker/cassandra']) ? metricsReport.metrics['broker/cassandra'].readBytes : 0,
+                        },
+
+                        startTime: metricsReport.startTime,
+                        currentTime: metricsReport.currentTime,
+                        timestamp: metricsReport.currentTime,
+                    }
+
+                    if (sec === 1) {
+                        minReport.peerName = secReport.peerName
```

Comment:
Instead of replacing values, why not just initialize the `minReport` object here?